<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoppingwidget</class>
 <widget class="QWidget" name="shoppingwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Place Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* Stile generale per il widget principale */
QWidget#shoppingwidget {
	background-color: #f0f0f0; /* Sfondo grigio chiaro */
	font-family: Segoe UI, Arial, sans-serif; /* Font pi\u00F9 moderno (se disponibile) */
	font-size: 10pt;
}

/* Stile per tutte le etichette */
QLabel {
	color: #333333; /* Testo grigio scuro */
	padding-bottom: 2px; /* Piccola spaziatura sotto */
}

/* Etichette descrittive (quelle a sinistra nel form) */
QLabel[objectName^="label"] {
	font-weight: bold;
	color: #555555; /* Grigio leggermente più chiaro per le label */
	padding-right: 10px; /* Spazio tra label e valore */
}

/* Etichette valore (quelle a destra nel form) */
QLabel[objectName^="value"] {
	color: #111111; /* Valori un po' più scuri */
}

/* Campo di testo per la descrizione */
QTextEdit#valueDesc {
	background-color: #ffffff;
	border: 1px solid #cccccc;
	border-radius: 4px;
	padding: 5px;
	color: #333333;
}

QTextEdit#valueDesc[readOnly="true"] {
	background-color: #f8f8f8; /* Sfondo leggermente diverso se read-only */
}

/* Stile per i bottoni (copiato da mainwindow.ui) */
QPushButton {
	background-color: #E53935; /* Rosso primario (Material Design Red 600) */
	color: white; /* Testo bianco */
	border: none; /* Nessun bordo */
	padding: 8px 16px; /* Pi\u00F9 padding */
	border-radius: 4px; /* Angoli arrotondati */
	font-weight: bold; /* Grassetto */
	min-width: 70px; /* Larghezza minima */
	margin: 2px; /* Piccolo margine per separare i bottoni */
}

QPushButton:hover {
	background-color: #D32F2F; /* Rosso pi\u00F9 scuro al passaggio (MD Red 700) */
}

QPushButton:pressed {
	background-color: #B71C1C; /* Rosso ancora pi\u00F9 scuro quando premuto (MD Red 900) */
}

/* Frame che contiene i dettagli (la &quot;card&quot;) */
QFrame#cardFrame {
	background-color: #ffffff; /* Sfondo bianco per la card */
	border: 1px solid #e0e0e0; /* Bordo molto sottile */
	border-radius: 6px; /* Angoli arrotondati */
}

/* Linea separatrice */
QFrame#separatorLine {
	background-color: #cccccc; /* Grigio per la linea */
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <!-- Modificato da QToolButton a QPushButton -->
      <widget class="QPushButton" name="pushEdit">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <!-- Modificato da QToolButton a QPushButton -->
      <widget class="QPushButton" name="pushDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="separatorLine">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="cardFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="cardLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="valueName">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelCity">
          <property name="text">
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="valueCity">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelDesc">
          <property name="text">
           <string>Description:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="valueDesc">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','Arial','sans-serif'; font-size:10pt;&quot;&gt;This is the description area.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelRating">
          <property name="text">
           <string>Rating:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="valueRating">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelCost">
          <property name="text">
           <string>Cost:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="valueCost">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelOpenings">
          <property name="text">
           <string>Openings:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="valueOpenings">
          <property name="text">
           <string>–</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <!-- Campi specifici per Shopping -->
        <item row="6" column="0">
         <widget class="QLabel" name="labelOutdoor">
          <property name="text">
           <string>Outdoor:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="valueOutdoor">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelFoodArea">
          <property name="text">
           <string>Food Area:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="valueFoodArea">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="labelStandNumber">
          <property name="text">
           <string>Stand Number:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="valueStandNumber">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
        <!-- Fine campi specifici per Shopping -->
        <item row="9" column="0">
         <widget class="QLabel" name="labelSpecific">
          <property name="text">
           <string>Specific:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="valueSpecific">
          <property name="text">
           <string>–</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="categoryLayout"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>